test/functional/lib/tests.cpp

```diff
@@ -6487,6 +6583,74 @@ GenericTxChecksumOffloadIp()
     MpTxFlush(GenericMp);
 }
 
+VOID
+GenericRxChecksumOffloadIp() {
```

Comment:
This test is written with my limited understanding of the AF_XDP api and how RX offloads ought to behave.